imap_processing/codice/codice_l1a.py

```diff
@@ -104,6 +105,11 @@ def create_science_dataset(self, start_time: np.datetime64) -> xr.Dataset:
         xr.Dataset
             ``xarray`` dataset containing the science data and supporting metadata
         """
+        cdf_attrs = ImapCdfAttributes()
+        cdf_attrs.add_instrument_global_attrs("codice")
+        cdf_attrs.add_instrument_variable_attrs("codice", "l1a")
+
+        # TODO: Is there a way to get the attrs from the YAML-based method?
```

Comment:
It wasn't quite clear to me how I might get the same information as is defined in `ConstantCoordinates.EPOCH` from the YAML files

Reply:
Yeah looking at it now I might have... removed it? Or something? I remember moving that over but it's not in the files... :( 
Anyway, it should probably be in either its own file or in some kind of default IMAP variable attrs yaml file.

Reply:
This is interesting. I looked in HERMES repo and having hard time finding it.

Reply:
Could this be what we need to use for `EPOCH`? https://github.com/IMAP-Science-Operations-Center/imap_processing/blob/dev/imap_processing/cdf/config/shared/default_variable_cdf_attrs_schema.yaml#L5

Reply:
Yeah that's the schema for it, but we still need to define the attribute somewhere. I'm not sure where it should live, maybe even in its own file? It's such a special case